Formula/signal-cli.rb

```diff
@@ -0,0 +1,24 @@
+class SignalCli < Formula
+  desc "CLI and dbus interface for WhisperSystems/libsignal-service-java"
+  homepage "https://github.com/AsamK/signal-cli"
+  url "https://github.com/AsamK/signal-cli/releases/download/v0.6.2/signal-cli-0.6.2.tar.gz"
+  sha256 "d917507211b3419e271674df7f03c8189921e2e78a389479a612847ddc8f1614"
+  depends_on :java => "1.7+"
```

Comment:
Blank line above `depends_on`

Reply:
Also please add a `bottle :unneeded` line

Reply:
Both done.

Reply:
There should be an empty line between bottle and depends_on too so almost there =)

Reply:
Added. :)